### ISSUE TITLE
aria-labelledby on object still throws error

### ISSUE THREAD

**Issue by robfentress** [dequelabs/axe-core#171]
This page incorrectly throws error:
[http://www.test.assist.vt.edu/sandbox/tester-tests/objectMustContainText-aria-labelledby.html](http://www.test.assist.vt.edu/sandbox/tester-tests/objectMustContainText-aria-labelledby.html)

The aria-labelledby attribute should be considered an adequate text alternative based on [your own explanation](https://dequeuniversity.com/rules/axe/1.1/object-alt) of "how to fix the problem".

**Comment by marcysutton** [comment]
Note that `aria-hidden` is affecting the check for `aria-labelledby` on an object tag.

**Comment by marcysutton** [comment]
Worth considering at the same time: https://github.com/dequelabs/axe-core/issues/170

**Comment by robfentress** [comment]
Indeed, using aria-hidden for the label would not be an unusual use case, I'd think.

**Comment by dylanb** [comment]
yes, `aria-labelledby` does not work on all elements - despite the ARIA specification saying that it should https://www.w3.org/TR/wai-aria/states_and_properties#aria-labelledby.

Now WCAG does have an "accessibility supported" normative requirement https://www.w3.org/TR/WCAG20/#glossary

Your page works in OS X with VoiceOver, iOS with VoiceOver, can someone confirm that this works with NVDA and FF; JAWS and IE; and with Talkback and Firefox?

**Comment by robfentress** [comment]
I can confirm that it works with JAWS 16 on FF and IE and NVDA 2015.4 with FF and IE

**Comment by robfentress** [comment]
Also works with Firefox on Android 5.1.1

**Comment by dylanb** [comment]
ok, well then this is definitely a bug :-) thanks

**Comment by robfentress** [comment]
Thank _you_. You folks are providing some incredible resources.  Keep up the good work!

**Comment by WilcoFiers** [comment]
We've created a patch in our internal repo. Should be good to go for the next release!
